src/graphql/data/nft/Collection.ts

```diff
@@ -87,26 +86,14 @@ const collectionQuery = graphql`
   }
 `
 
-export function useLoadCollectionQuery(address?: string | string[]): void {
-  const [, loadQuery] = useQueryLoader(collectionQuery)
-  useEffect(() => {
-    if (address) {
-      loadQuery({ addresses: Array.isArray(address) ? address : [address] })
-    }
-  }, [address, loadQuery])
-}
+export function useCollectionQuery(address: string): { data: GenieCollection; loading: boolean } {
```

Comment:
is there a helper type for gql-like return types like these? in mobile have a generic defined like `https://github.com/Uniswap/mobile/blob/main/src/features/dataApi/types.ts#L23` that defines data, loading, and error

Reply:
well, it prevents you from having to type out the whole object every time. you can just pass in `GenieCollection` as the type argument